compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -759,7 +759,7 @@ fn link_natively(
     sess.dcx().abort_if_errors();
 
     // Invoke the system linker
-    info!("{:?}", &cmd);
+    info!("{cmd:?}");
```

Comment:
```suggestion
    info!(?cmd);
```

Maybe it should be another clippy lint to turn to this?

Reply:
It's not exactly the same, the latter prints `cmd = xxzzzzz`